lib/std/http.zig

```diff
@@ -213,21 +218,22 @@ pub const Status = enum(u10) {
     }
 
     pub const Class = enum {
+        nonstandard,
         informational,
         success,
         redirect,
         client_error,
         server_error,
     };
 
-    pub fn class(self: Status) ?Class {
+    pub fn class(self: Status) Class {
         return switch (@enumToInt(self)) {
             100...199 => .informational,
             200...299 => .success,
             300...399 => .redirect,
             400...499 => .client_error,
             500...599 => .server_error,
-            else => null,
+            else => .nonstandard,
```

Comment:
Status codes outside of 100..599 are invalid, not nonstandard.

https://www.rfc-editor.org/rfc/rfc9110#section-15

> All valid status codes are within the range of 100 to 599, inclusive.

> Values outside the range 100..599 are invalid. Implementations often use three-digit integer values outside of that range (i.e., 600..999) for internal communication of non-HTTP status (e.g., library errors). A client that receives a response with an invalid status code SHOULD process the response as if it had a [5xx (Server Error)](https://www.rfc-editor.org/rfc/rfc9110#status.5xx) status code.